ee/tabby-schema/src/schema/web_documents.rs

```diff
@@ -0,0 +1,81 @@
+use async_trait::async_trait;
+use chrono::{DateTime, Utc};
+use juniper::{GraphQLInputObject, GraphQLObject, ID};
+use validator::Validate;
+
+use crate::{job::JobInfo, juniper::relay::NodeType, Context, Result};
+
+#[derive(GraphQLObject)]
+#[graphql(context = Context)]
+pub struct CustomWebDocument {
+    pub url: String,
+    pub name: String,
+    pub id: ID,
+    pub created_at: DateTime<Utc>,
+    pub job_info: JobInfo,
+}
+
+#[derive(GraphQLObject)]
+#[graphql(context = Context)]
+pub struct PresetWebDocument {
+    pub name: String,
+    pub id: ID,
+    pub job_info: JobInfo,
```

Comment:
missing `created_at`

Reply:
An in-active `PresetWebDocument` Does not have a create time.... I think we shall add update time instead

Reply:
good point - maybe `updated_at: Option<DateTime<Utc>>`